### ISSUE TITLE
Redirect is not triggered when $mail parameter is missing on success page

### ISSUE THREAD

**Issue by mawo** [in2code-de/powermail#876]
TYPO3 v11.5
Powermail v10.7.2

When accessing the "success" URL without mail data I expect a redirect (or forward) to the mail form.

Instead an error message is shown:
```
1298012500 TYPO3\CMS\Extbase\Mvc\Controller\Exception\RequiredArgumentMissingException
Required argument "mail" is not set for In2code\Powermail\Controller\FormController->create
```

The URL configuration looks like this (for the case it is a configuration issue at our side):
```yaml
routeEnhancers:
  Powermail:
    type: Extbase
    extension: Powermail
    plugin: Pi1
    routes:
      -
        routePath: /
        _controller: 'Form::form'
      -
        routePath: /gesendet
        _controller: 'Form::create'
      -
        routePath: /bestaetigung
        _controller: 'Form::confirmation'
```

When accessing the URL `/path/to/form/gesendet` the error mentioned above is shown.

The source code in `FormController` looks like the redirect _should_ be triggered. But it is not. see initializeCreateAction():

```php
    public function initializeCreateAction(): void
    {
        …
        $this->forwardIfMailParamEmpty();
        …
    }
```

The function `forwardIfMailParamEmpty()` returns a `ForwardResponse` but the response is never used.

```php
    protected function forwardIfMailParamEmpty(): ?ForwardResponse
    {
        $arguments = $this->request->getArguments();
        if (empty($arguments['mail'])) {
            $logger = ObjectUtility::getLogger(__CLASS__);
            $logger->warning('Redirect (mail empty)', $arguments);
            return new ForwardResponse('form');
        }
        return null;
    }
```

**Comment by dmnkhhn** [comment]
The redirect works if you replace `return new ForwardResponse('form');` with `$this->forward('form');`

**Comment by mawo** [comment]
I confirm, this is solved at least in Powermail 10.8.1
